Makefile

```diff
@@ -31,6 +31,10 @@ setup-dev: pipenv
 setup-no-flash: pipenv
 	$(PIPENV) install --verbose --python $(PYTHON_VERSION)
 
+setup-ui-test: pipenv
+	$(PIPENV) install --verbose --categories=dev-packages  
+	$(PIPENV) run playwright install 
```

Comment:
Could you also add playwright install to `setup-dev` so that `setup-dev` installs all dependencies needed for testing?